tests/infra/network.py

```diff
@@ -1098,22 +1099,12 @@ def wait_for_new_primary(
         error = TimeoutError
         logs = []
 
-        backup = self.find_any_backup(old_primary)
```

Comment:
BFT-specific code was removed here.

Reply:
Same as above, that would be better in its own PR, imo.